feature-libs/order/assets/translations/en/order.i18n.ts

```diff
@@ -0,0 +1,169 @@
+export const order = {
```

Comment:
Ideally we need to deprecate the corresponding i18n labels in @spartacus/assets.

We can't deprecate the whole `const myAccount` but perhaps we add deprecation comments on the sections that were copied here so we know what to delete when the time comes.